docs/API.Auth.ChangePassword.md

```diff
@@ -0,0 +1,37 @@
+# Function: *Change Password* #
+
+## Description ##
+
+Usually new users are assigned random passwords. After creating a CM-Well user, you may want to change the user's password to a customized value. You can do this with the **change-password**.
+
+>**Note:** You will need an access token with admin permissions to call this API.
+
+## Syntax ##
+
+**URL:** \<CMWellHost\>/_auth?op=change-password&current=$2a$10$7AnXsjks.IZXTbpRiAGN4OQItwiz4sgxM49lvTiCjWgOhbbOQkg2m&new=12345678 -H "X-CM-Well-Token:\<AdminToken\>"
+**REST verb:** GET
+**Mandatory parameters:** Admin-permissions token in the **X-CM-Well-Token** header
+
+## Special Parameters ##
+
+Parameter | Description 
+:---------|:-------------
+current   | The user's current password
+new       | The user's new password
+
+## Code Example ##
+
+### Call ###
+
+    <CMWellHost>/_auth?op=change-password&current=$2a$10$7AnXsjks.IZXTbpRiAGN4OQItwiz4sgxM49lvTiCjWgOhbbOQkg2m&new=12345678 -H "X-CM-Well-Token:<AdminToken>"
```

Comment:
same as above. for the "current" value - use the plain-text password, not its encrypted value